dirtree.go

```diff
@@ -4,14 +4,33 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	gitignore "github.com/sabhiram/go-gitignore"
 )
 
 func main() {
-	rootDir := "."
+	var rootDir string
+
+	switch len(os.Args) {
+	case 1:
+		rootDir = "."
+		fmt.Println(".")
+	case 2:
+		rootDir = strings.Join(os.Args[1:2], "")
+		_, err := os.Stat(rootDir)
+		if os.IsNotExist(err) {
+			fmt.Println("File does not exist")
```

Comment:
**Golang Best Practices:** Error strings should not be capitalized (unless beginning with an exported name, a proper noun or an acronym) and should not end with punctuation. This is because error strings usually appear within other context before being printed to the user.